### ISSUE TITLE
Data is updated only when Save button is pressed.

### ISSUE THREAD

**Issue by e-pirate** [mchugh19/zabbix-vue#21]
Extension retrieves actual data from the Zabbix server only once when Save button is pressed in the Server setup page and never again. No Problems are shown when they detected by Zabbix. When I then go to the extension configuration page, press Save again without any changes to the server settings, data gets updated once.

Confirmed by tcpdump showing no traffic exchange between host with extension and Zabbix server.

FF 60.3 for Linux.

**Comment by mchugh19** [comment]
Interesting! I've not been able to replicate this behavior. Might there be other plugins which are preventing the background.js script from running? 

With chrome, you can 'debug' a plugin which allows you to pull up the inspector screen for the plugin pages. From the javascript console, it allows you to see the various log messages begin generated by the background page. In this output, I see the plugin attempting to reconnect to the zabbix server on the defined schedule.

I don't see a way to access the console for background.js from firefox, so more research is necessary here for me to track this down.

**Comment by e-pirate** [comment]
Frankly speaking, I'm not able to reproduce it any more as well. I had two equal machines showing exactly same behaviour: first no traffic, just as I described in the bug, and now both working flawlessly. Originally I was blaming FoxyProxy, but one of the machines has it disabled. The other one now works with FoxyProxy with no problems. The only thing changed - I restarted FF. Both original tests were made after extension was installed, but FF wasn't restarted. That's my guess. I have no clue. Tell me what to do and I will try to help, but it working now :)
